src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.html

```diff
@@ -16,16 +10,16 @@ <h4 class="modal-title" jhiTranslate="entity.exportRepos.title">Confirm export o
             >
             <table class="table table-striped">
                 <tbody>
-                    <tr *ngFor="let programmingExercise of selectedProgrammingExercises">
+                    <tr *ngFor="let programmingExercise of programmingExercises">
                         <td>{{ programmingExercise.title }}</td>
                     </tr>
                 </tbody>
             </table>
         </div>
         <p
             *ngIf="!singleParticipantMode && !isRepoExportForMultipleExercises"
-            [jhiTranslate]="'artemisApp.instructorDashboard.exportRepos.questions.' + (exercise.teamMode ? 'team' : 'individual')"
-            [translateValues]="{ exerciseTitle: exercise.title, courseTitle: exercise.course?.title }"
+            [jhiTranslate]="'artemisApp.instructorDashboard.exportRepos.questions.' + (programmingExercises[0].teamMode ? 'team' : 'individual')"
```

Comment:
Why does this only refer to the first exercise in the array?

Reply:
As now the exercise (or exercises) is/are passed through a `programmingExercises` array, I need to reference the single downloaded exercise as first element of the array as there is only one in this case (see `*ngIf= ... !isRepoExportForMultipleExercises` at line 20).. 🙂